packages/core/admin/admin/src/content-manager/components/RelationInputDataManager/RelationInputDataManager.js

```diff
@@ -117,11 +119,15 @@ export const RelationInputDataManger = ({
   };
 
   const handleSearch = (term) => {
-    searchFor(term, { idsToOmit: modifiedData?.[name]?.connect?.map((relation) => relation.id) });
+    searchFor(term, {
+      idsToOmit: relationsFromModifiedData?.connect?.map((relation) => relation.id),
```

Comment:
I assume `idsToOmit` accepts `undefined` as an argument and doesn't require an `Array` of any length?

Reply:
indeed forgot about that, thank you!